src/index.ts

```diff
@@ -538,8 +543,11 @@ https://blog.newrelic.com/product-news/aws-lambda-extensions-integrations/
     }
   }
 
-  private async getLayerArn(runtime: string) {
-    const url = `https://${this.region}.layers.newrelic-external.com/get-layers?CompatibleRuntime=${runtime}`;
+  private async getLayerArn(runtime: string, architecture?: string) {
+    let url = `https://${this.region}.layers.newrelic-external.com/get-layers?CompatibleRuntime=${runtime}`;
+    if (architecture) {
+      url = `${url}&CompatibleArchitecture=${architecture}`;
```

Comment:
This won't work in regions that don't yet support ARM64. Recommend giving it a try with a region like `ap-northeast-2` and see how it doesn't behave the way you would expect.

Reply:
Ah, ok--that'll be a fix for Friday.

Reply:
@kolanos Interestingly, when I add ap-northeast-2 as a test case, querying the api works the way I'd expect, falling back to the x86 layer. Do we have a known-failing region?